niceman/distributions/redhat.py

```diff
@@ -0,0 +1,339 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the niceman package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Support for Redhat(-based) distribution(s)."""
+import attr
+
+from six.moves import map
+
+import logging
+
+from niceman.distributions.base import DistributionTracer
+
+lgr = logging.getLogger('niceman.distributions.redhat')
+
+from .base import SpecObject
+from .base import Package
+from .base import Distribution
+from .base import TypedList
+from .base import _register_with_representer
+from ..support.exceptions import CommandError
+
+
+@attr.s(cmp=True)
+class RPMSource(SpecObject):
+    """RPM origin information"""
+    id = attr.ib()
```

Comment:
now that #234 is merged, please adjust to use "our" attrib

Reply:
I merged the latest master which converted all attr.ib() to attrib()